test/plugin/scenarios/httpasyncclient-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+registryItems:
+    applications:
+    - {httpasyncclient-scenario: nq 0}
+    instances:
+    - {httpasyncclient-scenario: 1}
+    operationNames:
+    - httpasyncclient-scenario: [/httpasyncclient/case/httpasyncclient, /httpasyncclient/back]
+segmentItems:
+- applicationCode: httpasyncclient-scenario
+  segmentSize: 4
```

Comment:
Suggestion, `ge 4`
Sometimes, this will be called several times.

Reply:
Yes, agree. `ge 4` is more safe. @aderm Fix this?

Reply:
ok